README.md

```diff
@@ -329,9 +330,9 @@ For building an image based on DSE 6.9, see the [DSE 6.9 README](management-api-
 By default, all images will listen on port 8080 for Management API connections. This can be overridden by specifying
 the environment variable `MGMT_API_LISTEN_TCP_PORT` and setting it to your desired port. For example:
 
-    > docker run -e USE_MGMT_API=true -e MGMT_API_LISTEN_TCP_PORT=9090 -p 9090:9090 k8ssandra/cass-management-api:4.0.14
+    > docker run -e USE_MGMT_API=true -e MGMT_API_LISTEN_TCP_PORT=9090 -p 9090:9090 k8ssandra/cass-management-api:4.0.15
 
-The above would run a Cassandra 4.0.14 image with Management API listening on port 9090 (instead of 8080).
+The above would run a Cassandra 4.0.15 image with Management API listening on port 9090 (instead of 8080).
```

Comment:
**nit:**
You don't have to change this every time. This is just a README example.